webrtc.html

```diff
@@ -4681,6 +4681,25 @@ <h2>RTP Media API</h2>
     added to an <code>RTCPeerConnection</code>, result in signaling; when this
     signaling is forwarded to a remote peer, it causes corresponding tracks to
     be created on the remote side.</p>
+    <p class="note">There is not an exact 1:1 correspondence between tracks sent
+    by one <code>RTCPeerConnection</code> and received by the other. For one,
+    in many cases the ID of a track sent by one side will not match the ID of
+    the corresponding track received on the other side; whether the IDs end up
+    matching depends on the relative ordering of calls to
+    <code><a data-link-for="RTCPeerConnection">addTrack</a></code>,
+    <code><a data-link-for="RTCPeerConnection">addTransceiver</a></code> and
+    <code><a data-link-for="RTCPeerConnection">setRemoteDescription</a></code>,
+    and which side generates the offer. Also, if
+    <code><a data-link-for="RTCRtpSender">replaceTrack</a></code> is called,
+    changing the track sent by an <code><a>RTCRtpSender</a></code>, no new
+    track will be created on the receiver side; the corresponding
+    <code><a>RTCRtpReceiver</a></code> will only have a single track,
+    potentially representing multiple sources of media stiched together. Thus
+    it's more accurate to think of a 1:1 relationship between
+    <code><a>RTCRtpSender</a></code>s on one side and
+    <code><a>RTCRtpReceiver</a></code>s on the other side, matching them using
+    the <code><a>RTCRtpTransceiver</a></code>'s <code><a
+    data-link-for="RTCRtpTransceiver">mid</a></code> if necessary.</p>
```

Comment:
The transceiver on one side maps 1:1 with the transceiver on the other end with the same "mid". The sender's track is what is sent to the receiver's track. The sender's track can be replaced or nulled, which affects what media shows up in the receiver's track. The receiver's track never changes.